templates/experimental/non_opinionated/README.md

```diff
@@ -0,0 +1,74 @@
+# expressots-experimetal-non_opinionated
+
+The expressots-experimetal-non_opinionated project is an initiative that seeks to explore the possibilities of a faster and more efficient build system for the expressots framework. Expressots is a TypeScript-based framework that facilitates the development of robust and scalable web applications. The experimental build system of the expressots-experimetal-non_opinionated project promises to be much more performant than the current version of expressots, but it is still in the testing and validation phase. Therefore, it is not recommended to use this project in production environments, as there may be bugs and inconsistencies. The expressots framework website (https://expresso-ts.com/) offers a detailed documentation on how to use expressots and its features.
+
+## Server
+The server runs on the http://localhost:3000. Opening this url, the server response is `Hello Expresso TS!`. This means that the server is working properly and ready to handle requests.
+
+## Installation
+
+To install the project dependencies, run the following command:
+
+```bash
+npm i
```

Comment:
Is it possible to use other package managers?

Reply:
Well, we know the package manager at the CLI creation, we could generate this `README` with the correct package manager indeed.

Reply:
But I think this is room for improvement in another issue